src/utils.ts

```diff
@@ -529,6 +541,20 @@ export class Utils {
     (target || e.target).dispatchEvent(simulatedEvent);
   }
 
+  public static getNewXYPosition(event: MouseEvent, element: GridItemHTMLElement) {
+    const containerElement = Utils.getContainerElement(element.parentElement);
+    const containerBoundingClientRect = containerElement.getBoundingClientRect();
+    const boundingClientRect = element.getBoundingClientRect();
+    const scaleX = boundingClientRect.width / element.offsetWidth;

```

Comment:
woudl that work is scaling in set multiple times on different parents and not just immediate one ?

Reply:
Yes it will :) !